README.md

```diff
@@ -23,14 +23,75 @@ practices.
 ## Versioning
 The commit hash of the deployed code is considered its version identifier. The commit hash can be retrieved locally via `git rev-parse HEAD`.
 
-## Setting Up
+## Setting Up Your Local Dev Environment
 
 Contile uses Rust, and requires the latest stable iteration. See
 [rustup.rs](https://rustup.rs/) for how to install this application.
 
 Once Rust is installed you can compile using `cargo build`. This will
 create a development release.
 
+Additionally, Contile requires a connection to the ADM servers. These use several 
+keys (which are specified in the configuration file) to pull results.  The contract
+tests include a stub application that can provide ADM-like returns. See the 
+[adm_settings_test.json](./adm_settings_test.json) file for reference.
+
+Normally, these configurations run as a part of a `docker-compose` command, but the individual
+Docker files can be run by themselves.
+
+To start the ADM stub, you can run the following from the root of the Contile source repository:
+
+```shell
+    docker run \
+    --env PORT=5000 \
+    --env RESPONSES_DIR=/tmp/partner/ \
+    --env ACCEPTED_MOBILE_FORM_FACTORS=phone,tablet \
+    --env ACCEPTED_DESKTOP_FORM_FACTORS=desktop \
+    -v `pwd`/test-engineering/contract-tests/volumes/partner:/tmp/partner \
+    -p 5000:5000 \
+    mozilla/contile-integration-tests-partner
+```
+
+To use the above, you'll need to set the Contile configuration option as follows:
+`adm_endpoint_url`  = `http://localhost:5000/tilesp/desktop`.  Note, you also need to 
+define a SOV (Share of Voice) source file, so make sure this is provided for partner allocations. Currently, a sample file is in the contract-tests directory, so you can make use of that, or define your own allocation JSON file to point to. 
+
+It may be helpful to create a settings file to contain all of the preferred settings, like the example of `sa-test.toml` below:
```

Comment:
Just a thought on explanations: I would suggest that we reduce the amount of explanatory text for set up documentation. There's a few reasons for this:

1. Audience: users who use set up documentation usually have the goal to set up their service and care less about _how_ the pieces fit together. So a lot of this explanatory text will get in the way of their goal.
2. Explanatory text here will introduce another surface area for documentation maintenance (i.e. if we change Share of Voice related setup, then we'll need to change the explanation here too)
3. Giving users _options_ for set up will actually be more confusing and make it longer for users to actually get something set up properly.

The [Application Service Build set up](https://mozilla.github.io/application-services/book/building.html#macos) is a good example of how we should write set up documentation. Some things to note:

1. Code commands can be copy and pasted directly into the terminal
2. Instructions are short and straight to the point (don't explain the actual set up or the reason for requiring it, as it's not important to getting it actually set up).
3. Sequencing for steps are clear and numbered. If users have trouble, it's much easier to ask for help with the reference "Step X" rather than "the point I install XYZ".

Reply:
Sounds good @mtbottle , I agree.  Shall I update the runbook to match the same tone and reduce verbiage so the two match? I was going off of it for this update, so it's probably best if the match, right?

Reply:
Ah yes! The original runbook should also be updated to match! Thanks for fixing that up as well!